iteration/iteration.go

```diff
@@ -253,6 +256,37 @@ func (m *GormIterationRepository) CanStart(ctx context.Context, i *Iteration) (b
 	return true, nil
 }
 
+func inTimeframe(startAt time.Time, endAt time.Time) bool {
+	if time.Now().UTC().After(startAt) && time.Now().UTC().Before(endAt) {
```

Comment:
This could technically just be:

    return time.Now().UTC().After(startAt) && time.Now().UTC().Before(endAt)

Reply:
handled it here -> fa40fe1